style/rdg-cell.less

```diff
@@ -149,3 +149,21 @@
   cursor: help;
   padding-left: 4px;
 }
+
+.rdg-dragging {
+  cursor: grabbing;
+}
+
+.rdg-draggable-header-cell {
+  cursor: move;
+  display: inline-block;
+  height: inherit;
+  background-color: inherit;
+  width: 100%;
+  margin-left: -8px;
```

Comment:
This will affect the dragging preview. Without this, the preview text will have no left space in the preview container. Maybe we can revisit the `header` css together later.